skimage/_shared/utils.py

```diff
@@ -415,3 +418,12 @@ def _validate_interpolation_order(image_dtype, order):
              FutureWarning, stacklevel=2)
 
     return order
+
+
+def _fix_ndimage_mode(mode):
+    # SciPy 1.6.0 introduced grid variants of constant and wrap which
+    # have less surprising behavior for images. Use these when available
+    grid_modes = {'constant': 'grid-constant', 'wrap': 'grid-wrap'}
+    if NumpyVersion(scipy.__version__) >= '1.6.0' and mode in grid_modes:
```

Comment:
As for the other PR, I think a bullet in the todo list for next year would be helpful. :)

Reply:
Sure. I just reorganized that TODO section in #5173, though, so I think I will wait until we merge that one and then rebase here and add it

Reply:
I rebased now that #5173 was merged and a TODO has been added